docs/conf.py

```diff
@@ -122,6 +130,7 @@
 # expandvars
 expandvars_define = {
     "PYDIR": "frozen" if RTD_BUILD else "src",
+    #"PYDIR": "frozen",
```

Comment:
Unintentional commented line.

Reply:
Should I remove that line?

Reply:
Yes, please. GitHub is showing that you added it. It doesn't seem to serve a purpose.

Reply:
Done

Reply:
Thanks Craig.  We are waiting on the second approval and then should be able to merge.